docs/DRONECAN.md

```diff
@@ -0,0 +1,38 @@
+# mLRS Documentation: DroneCAN for ArduPilot Systems #
+
+([back to main page](../README.md))
+
+DroneCAN can be utilized for MAVLink and RC with mLRS instead of traditional serial communication if both the flight controller and receiver have the hardware necessary to support it.
+
+## Wiring
+
+TBD
+
+## ArduPilot Settings
+
+The following parameters need to be set to enable MAVLink and RC over DroneCAN.
+
+Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+
+- CAN_P1_DRIVER = 1
+- CAN_D1_PROTOCOL = 1
```

Comment:
maybe one could add that it's 1 by default alread
like
- CAN_D1_PROTOCOL = 1 (normally set to 1 by default)

Reply:
Sure, done